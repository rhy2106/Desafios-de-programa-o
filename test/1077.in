6
a+b/(c*d^e)
((a+b)*C-(D-e))^(f-g)
A^B*C+E/F/(G-H)
A*2
(A*2+c-d)/2
(2*4/a^b)/(2*c)