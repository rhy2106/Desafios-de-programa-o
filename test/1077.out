abcde^*/+
ab+C*De--fg-^
AB^C*EF/GH-/+
A2*
A2*c+d-2/
24*ab^/2c*/
